apps/src/templates/sectionProgress/progressTables/ProgressTableContainer.story.jsx

```diff
@@ -12,86 +14,116 @@ import {createStore, wrapTable} from '../sectionProgressTestHelpers';
  */
 const INCLUDE_LARGE_STORIES = false;
 
-function buildSmallStories(component, currentView) {
+class _TableWrapper extends React.Component {
+  static propTypes = {
+    currentView: PropTypes.oneOf(Object.values(ViewType))
+  };
+  render() {
+    return (
+      <div
+        className="main"
+        style={{
+          marginLeft: 80,
+          width: 970,
+          display: 'block',
+          backgroundColor: '#ffffff'
+        }}
+      >
+        <SectionProgressToggle />
```

Comment:
testing this work required being able to toggle between view types, hence this update to the storybook. this update is not directly related to the purpose of this PR.